src/core/aws-utils.ads

```diff
@@ -94,6 +101,11 @@ package AWS.Utils is
                          => C in '0' .. '9' | 'a' .. 'z' | 'A' .. 'Z');
    --  Returns random string
 
+   procedure Random_Reset (Seed : Integer);
+   --  This function need only if user want to get predictable random numbers.
+   --  It mean that after call to Random_Reset with same Seed the same sequence
```

Comment:
It means that after calling Random_Reset with the same seed the same sequence
of Random and Random_String calls will give the same results.

Reply:
done